ansible/inventory/test

```diff
@@ -66,6 +62,9 @@ devel-small-04.aws-eu-central-1a.nimbus.pyrmont
 libp2p-small-01.aws-eu-central-1a.nimbus.pyrmont
 libp2p-small-02.aws-eu-central-1a.nimbus.pyrmont
 
+[nimbus-build-severs]
+master-large-01.aws-eu-central-1a.nimbus.pyrmont
```

Comment:
You're missing master-small-01 which handles the builds for all \*-small-\* hosts.

Reply:
You can't edit this file by hand. Look at the top:
```
# NOTE: This file is generated by terraform.py
# For emergency use when Consul fails
```